src/coreneuron/io/nrn_setup.cpp

```diff
@@ -636,14 +636,14 @@ void read_phasegap(NrnThread& nt, UserParams& userParams) {
 // only voltage, i_membrane_ or mechanism data index allowed. (mtype 0 means time)
 double* stdindex2ptr(int mtype, int index, NrnThread& nt) {
     if (mtype == voltage) {  // voltage
-        int ix{index};       // relative to _actual_v
+        int ix = index;      // relative to _actual_v
```

Comment:
Just curious. What is the rationale for not using the initializer form?

Reply:
No need to revert. In the past I always used ```int foo = bar```. I just started seeing a lot of int foo{bar} in the c++ code and wondered if there was something going on that I wasn't aware of. I wondered if ```int foo{literal}``` is somehow better than ```int foo = literal``` because of moving an operation from runtime to compile time. But supposed that optimization would make them equivalent anyway.